repository-data/application/src/main/resources/hcm-config/namespaces/hee/link.yaml

```diff
@@ -40,6 +40,15 @@ definitions:
             hipposysedit:path: hee:document
             hipposysedit:primary: false
             hipposysedit:type: hippo:mirror
+          /openLinkUrlNewWindow:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: false
+            hipposysedit:ordered: false
+            hipposysedit:path: hee:openLinkUrlNewWindow
+            hipposysedit:primary: false
+            hipposysedit:type: Boolean
+            hipposysedit:validators: [required]
```

Comment:
Hi @Raq-HEE - Any reason why we kept `Open link url in new window` (`hee:openLinkUrlNewWindow`) field as mandatory ? thanks